server/utils/commonUtils.js

```diff
@@ -1,7 +1,7 @@
 const crypto = require('crypto');
 
 function hashPassword(password) {
-  const secret = process.env.HASH_SALT || "I'm a teapot";
+  const secret = "I'm a teapot";
   return crypto
```

Comment:
I think we still want this to be set to an env var. Looks like its a constant right now?

Reply:
you are absolutely right - dont know how that got dropped. Pushing it up now